gratipay/security/crypto.py

```diff
@@ -61,3 +64,47 @@ def constant_time_compare(val1, val2):
     for x, y in zip(val1, val2):
         result |= ord(x) ^ ord(y)
     return result == 0
+
+
+# Encrypting Packer
+# =================
+
+class EncryptingPacker(object):
+    """Implement conversion of Python objects to/from encrypted bytestrings.
+
+    :param str key: a `Fernet`_ key to use for encryption and decryption
+    :param list old_keys: additional `Fernet`_ keys to use for decryption
+
+    .. note::
+
+        Encrypted messages contain the timestamp at which they were generated
+        *in plaintext*. See `our audit`_ for discussion of this and other
+        considerations with `Fernet`_.
+
+    .. _Fernet: https://cryptography.io/en/latest/fernet/
+    .. _our audit: https://github.com/gratipay/gratipay.com/pull/3998#issuecomment-216227070
+
+    """
+
+    def __init__(self, key, *old_keys):
```

Comment:
Hmm, maybe this was done to emphasize the answer to https://github.com/gratipay/gratipay.com/pull/3998/files#r62548400?